crates/compiler/builtins/bitcode/src/list.zig

```diff
@@ -803,9 +803,9 @@ pub fn listConcat(list_a: RocList, list_b: RocList, alignment: u32, element_widt
     @memcpy(target, source_a, list_a.len() * element_width);
     @memcpy(target + list_a.len() * element_width, source_b, list_b.len() * element_width);
 
-    // decrement list a and b.
-    utils.decref(source_a, list_a.capacity, alignment);
-    utils.decref(source_b, list_b.capacity, alignment);
+    // deinit list a and b.
+    source_a.deinit(alignment);
+    source_b.deinit(alignment);
 
     return output;
 }
```

Comment:
You also will need to update the tests at the end of the file from `deinit(u8)` to `deinit(alignOf(u8))`

... I just can't leave a comment on that line because github only lets me leave comments on lines you editted.